docbot/main.go

```diff
@@ -311,6 +311,11 @@ func (a *Action) onPullRequestOpenedOrEdited() error {
 
 	// Add missing label
 	if a.config.GetEnableLabelMissing() && checkedCount == 0 {
+		if _, exist := currentLabelsSet[a.config.GetLabelMissing()]; exist {
+			logger.Infoln("Already added missing label.")
+			return fmt.Errorf("%s", MessageLabelMissing)
```

Comment:
```suggestion
			return nil;
```

Reply:
We should return the error here. Still the pr needs a valid doc label. We just skip the create comment step.